test/permissions/remote.provider.get.js

```diff
@@ -0,0 +1,191 @@
+'use strict';
+
+const jstp = require('@metarhia/jstp');
+
+const { codes: errorCodes } = require('../../lib/errors');
+
+const test = (test, { categories, users, roles, data, permissions }) => {
+  const expectedErrorCode = errorCodes.INSUFFICIENT_PERMISSIONS;
+  const expectedError = new jstp.RemoteError(expectedErrorCode);
+
+  test.endAfterSubtests();
+
+  test.test(
+    'remoteProvider.get: check read permission, sufficient permission',
+    (test, { remoteProvider, login }) =>
+      login(users.S1.ReadOnlyUser, () =>
+        remoteProvider.get(categories.Role, (err, res) => {
+          if (err) {
```

Comment:
Why not just `test.error`?

Reply:
When I started writing these tests, I come across with a problem, that in case of error this check
ended up not correctly, so I have to use `test.fail()` instead.

https://github.com/metarhia/metatests/blob/efc89b4dde7d9ef5e73a3a9cb54d500ec6881dcc/lib/imperative-test.js#L298

However, I check this now, and everything works correctly. Maybe missed smth, I will change it, thank you